zmq-sys/Cargo.toml

```diff
@@ -14,9 +14,11 @@ build = "build/main.rs"
 links = "zmq"
 
 [features]
+libsodium = ["dep:libsodium-sys-stable"]
 
 [dependencies]
 libc = "0.2.15"
+libsodium-sys-stable = { version = "1.0", optional = true }
```

Comment:
Because of semver, `1.0` really means `>=1.0.0`, which includes `1.18.1` and above.

That said it might make sense to validate that things actually build with that version, with i.e. `cargo +nightly -Zminimal-versions generate-lockfile`.

Reply:
Ok, I was not aware of that, thanks for educating me!

I wonder why changing the version number to something explicitly in their version list seemed to make a difference on my local setup though? I can no longer recall how things were failing (build issues vs. issues with my runtime testing). I was pretty sure that this change actually resolved whatever the issue was, but perhaps I changed multiple things... I simply don't remember anymore.